### ISSUE TITLE
Multi-Hop Support

### ISSUE THREAD

**Issue by kkhan01** [randymorris/tramp-term.el#15]
Often, many networks are setup in a manner such as there is a gateway server and from that you can ssh into 'local machines'. On command, it's possible to do multi hop in various way, one I use is:
```bash
ssh -tt $USER@$NETWORK -tt $MACHINE
```
In TRAMP, there are also a couple of ways, one of the popular ones outlined [here](https://www.gnu.org/software/emacs/manual/html_node/tramp/Ad_002dhoc-multi_002dhops.html). 

I mentioned both ways because it seems like this project kind of just opens an ansi-term and then does the ssh (I could be wrong, not wholly familiar with the code). I think it'd be better to use an already open TRAMP session if possible, eg if I'm on a tramp file like `~/someproject/hello.c` and invoke the function, it should open the terminal in `~/someproject` without requiring any user input (in a sense automatic).  But these are just extra features.

**TLDR**: It'd be cool to add support for multi hopping.
